install_config/router/customized_haproxy_router.adoc

```diff
@@ -199,16 +624,43 @@ After this modification, you can xref:rebuilding-your-router[rebuild your router
 [[rebuilding-your-router]]
 == Rebuilding Your Router
 
-After you have made any desired modifications to the template, such as the
-example xref:using-stick-tables[stick tables] customization, you must rebuild
-your router for your changes to go in effect:
-
-. https://access.redhat.com/articles/881893#createimage[Rebuild the container
-image to include your customized template.]
-. xref:../registry/accessing_registry.adoc#access[Push the resulting image to your repository].
-. Create the router specifying your new image, either:
-.. in the pod's object definition directly, or
-.. by adding the `--images=<repo>/<image>:<tag>` flag to the `oadm router`
-command when
-xref:../../admin_guide/high_availability.adoc#configuring-a-highly-available-service[creating
-a highly-available routing service].
+In order to rebuild the router, you need copies of several files that are present
+on a running router. Make a work directory and copy the files from the router:
+
```

Comment:
The public github openshift/origin repo will have a copy of each version that has been committed but it is not easy to find the versions of the shipped files. openshift/OSE has the files that are being shipped for each release. We provide a source RPM for the release (I think). We don't allow access to the openshift/OSE git repo. 

I personally found it easier to just pull them from the running pod.